File: typescript/src/_copy_me/Greeter.test.ts
import { Greeter } from "./Greeter";

describe(Greeter, () => {
  it("should return a message with 'Hello {name}'", () => {
    // arrange
    const greeter = new Greeter();

    // act
    const result = greeter.greet("Mauro");

    // assert
    // expect(result).toBe("Hello Mauro");
  });
});

File: java/src/test/java/_copy_me/GreeterTest.java
package rename_me;

import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.extension.ExtendWith;
import org.mockito.InjectMocks;
import org.mockito.Mock;
import org.mockito.junit.jupiter.MockitoExtension;

import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.core.Is.is;
import static org.mockito.BDDMockito.given;

@ExtendWith(MockitoExtension.class)
class GreeterTest {
    @Test
    void greet_should_return_Hello_name() {
        // arrange
        var greeter = new Greeter();

        // act
        var result = greeter.greet("Mauro");

        // assert
        // assertThat(result, is("Hello Mauro"));
    }
}
